hmftools.wdl

```diff
@@ -0,0 +1,927 @@
+        purpleQc: {description: "The QC file generated by purple.", category: "required"}
+        purpleDriverCatalogSomatic: {description: "The somatic driver catalog generated by purple.", category: "required"}
+        purpleDriverCatalogGermline: {description: "The germline driver catalog generated by purple.", category: "required"}
+        purpleSomaticVariants: {description: "The somatic VCF generated by purple.", category: "required"}
+        purpleSomaticVariantsIndex: {description: "The index for the somatic VCF generated by purple.", category: "required"}
+        purpleGermlineVariants: {description: "The germline VCF generated by purple.", category: "required"}
+        purpleGermlineVariantsIndex: {description: "The index of the germline VCF generated by purple.", category: "required"}
+        purpleGeneCopyNumber: {description: "The gene copy number file generated by purple.", category: "required"}
+        linxFusion: {description: "The fusion file generated by linx.", category: "required"}
+        linxBreakend: {description: "The breakend file generated by linx.", category: "required"}
+        linxDriversCatalog: {description: "The driver catalog generated generated by linx.", category: "required"}
+        chordPrediction: {description: "The chord prediction file.", category: "required"}
+        annotatedVirus: {description: "The virus-interpreter output.", category: "required"}
+
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        javaXmx: {description: "The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.",
+                  category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task Purple {
+    input {
+        String normalName
+        String tumorName
+        String outputDir = "./purple"
+        Array[File]+ amberOutput
+        Array[File]+ cobaltOutput
+        File gcProfile
+        File somaticVcf
+        File germlineVcf
+        File filteredSvVcf
+        File? fullSvVcf
+        File? fullSvVcfIndex
+        File referenceFasta
+        File referenceFastaFai
+        File referenceFastaDict
+        File driverGenePanel
+        File somaticHotspots
+        File germlineHotspots
+        
+        Int threads = 1
+        Int timeMinutes = 30
+        String memory = "9G"
+        String javaXmx = "8G"
+        # clone of quay.io/biocontainers/hmftools-purple:3.1--hdfd78af_0 with 'ln -s /usr/local/lib/libwebp.so.7 /usr/local/lib/libwebp.so.6'
+        String dockerImage = "quay.io/biowdl/hmftools-purple:3.1" 
```

Comment:
No availability on bioconda?

Reply:
See the comment above, I believe I was having a hard time getting this to work in conda.